bin/merge

```diff
@@ -34,8 +34,8 @@ directory=$(mktemp -d "/tmp/slamdata-merge.XXXXXXXX")
 
 # FIXME: removed -e around this command, don’t know why I had to
 set +e
-read -d '' merged title changelog base_clone pr_clone submitter base_sha base_ref pr_ref \
-     < <(curl -f -s $pr_url | json merged title body base.repo.ssh_url head.repo.ssh_url user.login base.sha base.ref head.ref -e 'this.body = (this.body || "").replace(/\r\n/g, "{{[[ENDL]]}}")')
+read -d '' merged title base_clone pr_clone submitter base_sha base_ref pr_ref changelog \
+     < <(curl -f -s $pr_url | json merged title base.repo.ssh_url head.repo.ssh_url user.login base.sha base.ref head.ref body -e 'this.body = this.body.replace(/\r\n/g, "{{[[ENDL]]}}")')
```

Comment:
I just moved the problematic value to the end, so it doesn’t off-by-1 the rest of the variables if it’s missing.